djvm/src/test/java/net/corda/djvm/Utilities.java

```diff
@@ -0,0 +1,23 @@
+package net.corda.djvm;
+
+import sandbox.net.corda.djvm.costing.ThresholdViolationError;
+import sandbox.net.corda.djvm.rules.RuleViolationError;
+
+/**
+ * Pin this {@link Utilities} class inside the sandbox to allow
+ * tests to invoke these functions.
+ */
+public final class Utilities {
```

Comment:
It is hard to distinguish the changes that are actually aligned with the purpose of the PR as stated by its title from the little "en-passant" fixes and refactoring

Reply:
This wasn't an "en passant" fix. The `Utilities` class and a few others need to be implemented in Java now instead of Kotlin so that the Java tests can load them _without_ the Kotlin libraries being present on the classpath :grinning:.